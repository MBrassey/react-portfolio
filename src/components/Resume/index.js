import FadeIn from "react-fade-in";
import React from "react";

function Resume() {
  return (
    <FadeIn>
      <section className="my-5">
        <div className="my-2">
          <br></br>
          <div className="resume-text">
            {" "}
            My past six years of expreience in the areas of:{" "}
            <strong>Git</strong>,<strong> Bash</strong>,
            <strong> Markdown</strong>,<strong> HTML5</strong>,
            <strong> CSS3</strong>, <strong> JS[ES6]</strong>,
            <strong> Node</strong>, <strong> MySQL</strong>,
            <strong> MongoDB</strong>, <strong> Express</strong>,
            <strong> React</strong>, <strong>JSX</strong> &
            <strong> Solidity</strong> are documented and can be reviewed on my{" "}
            <a href="https://github.com/MBrassey">github</a> profile. I recently
            obtained the{" "}
            <strong>
              <a href="https://www.credential.net/44a73119-b770-4a10-901c-7aa6508d5c65">
                Certified Blockchain Architect
              </a>
            </strong>{" "}
            credential from the <strong>Blockchain Council</strong>, as well as
            a{" "}
            <strong>
              {" "}
              <strong>
                <a href="https://raw.githubusercontent.com/MBrassey/MediaStore/main/Images/UofA_Graduate.png">
                  Full Stack Development Certification
                </a>
              </strong>{" "}
            </strong>
            from the <strong>University of Arizona</strong> for completing a 24
            Week Intensive Coding Bootcamp. I'm currently open to and actively
            seeking work in the areas of front/back end web & blockchain
            development. My resume is provided below, along with a chart
            depicting my past month's coding activity and badges representing my
            areas of proficiency.{" "}
          </div>
          <form
            action="https://docs.google.com/document/d/1T7uHv2RcH_wzERwJKoahqh9ODKdaJ48ci40nKEBViNY/"
            method="get"
            target="_blank"
          >
            <button alt="Download My Resume" type="submit">
              Download My Resume
            </button>
          </form>{" "}
          <br />
          <div className="resume-text">
            "Matt is a hard worker and was able to produce great code in a
            neat and timely manner. His contributions to our projects were
            beyond valuable, he shows a great understanding of the technologies
            we used within our full stack web app." -{" "}
            <a href="https://www.linkedin.com/in/ryan-brown-83760479/">
              Ryan Brown
            </a>
          </div>
          <br />
          <div className="resume-text">
            "It was awesome to watch the project come together thanks to Matt's
            ability to translate wireframed concepts in to reality, and deep
            knowledge of the technologies our app is built upon." -{" "}
            <a href="https://www.linkedin.com/in/matthew-ondrovic-a43826131/">
              Matthew Ondrovic
            </a>
          </div>
          <div className="CAcontainer">
            <div className="CodingActivity"></div>
          </div>
          <ul className="dev-icons">
            <li className="list-inline-item">
              <i className="fab fa-html5"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-css3-alt"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-bootstrap"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-markdown"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-ethereum"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-js-square"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-react"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-node-js"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-npm"></i>
            </li>
            <li className="list-inline-item">
              <i className="fab fa-git"></i>
            </li>
          </ul>
        </div>
      </section>
    </FadeIn>
  );
}

export default Resume;
