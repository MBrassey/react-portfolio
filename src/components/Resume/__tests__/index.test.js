import { render, cleanup, getByText, getByAltText } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import Resume from '../';

afterEach(cleanup);

describe('Resume component', () => {
  it('renders', () => {
    render(<Resume />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Resume />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('Certification Link 1 is available', () => {
    const { getByText } = render(<Resume />);
    const CertificationLink = getByText(/Certified Blockchain Architect/i);
    expect(CertificationLink).toBeInTheDocument();
    expect(CertificationLink).toHaveAttribute("href", "https://www.credential.net/44a73119-b770-4a10-901c-7aa6508d5c65");
  })
})
