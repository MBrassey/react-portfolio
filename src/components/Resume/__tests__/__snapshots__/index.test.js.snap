// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      style="transition: opacity 400ms, transform 400ms; transform: translateY(20px); opacity: 0;"
    >
      <section
        class="my-5"
      >
        <div
          class="my-2"
        >
          <br />
          <div
            class="resume-text"
          >
             My past six years of expreience in the areas of: 
            <strong>
              Git
            </strong>
            ,
            <strong>
               Bash
            </strong>
            ,
            <strong>
               Markdown
            </strong>
            ,
            <strong>
               HTML5
            </strong>
            ,
            <strong>
               CSS3
            </strong>
            , 
            <strong>
               JS[ES6]
            </strong>
            ,
            <strong>
               Node
            </strong>
            , 
            <strong>
               MySQL
            </strong>
            ,
            <strong>
               MongoDB
            </strong>
            , 
            <strong>
               Express
            </strong>
            ,
            <strong>
               React
            </strong>
            , 
            <strong>
              JSX
            </strong>
             &
            <strong>
               Solidity
            </strong>
             are documented and can be reviewed on my 
            <a
              href="https://github.com/MBrassey"
            >
              github
            </a>
             profile. I recently obtained the 
            <strong>
              <a
                href="https://www.credential.net/44a73119-b770-4a10-901c-7aa6508d5c65"
              >
                Certified Blockchain Architect
              </a>
            </strong>
             credential from the 
            <strong>
              <a
                href="https://www.blockchain-council.org"
              >
                Blockchain Council
              </a>
            </strong>
            , as well as a 
            <strong>
              Full Stack Development Certification 
            </strong>
            from the 
            <strong>
              University of Arizona
            </strong>
             for completing a 24 Week Intensive Coding Bootcamp. I'm currently open to and actively seeking work in the areas of front/back end web & blockchain development. My resume is provided below, along with a chart depicting my past week's coding activity and badges representing my areas of proficiency. 
          </div>
          <form
            action="https://brassey.io/resume.pdf"
            method="get"
            target="_blank"
          >
            <button
              alt="Download My Resume"
              type="submit"
            >
              Download My Resume
            </button>
          </form>
          <div
            class="CAcontainer"
          >
            <div
              class="CodingActivity"
            />
          </div>
          <ul
            class="dev-icons"
          >
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-html5"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-css3-alt"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-bootstrap"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-markdown"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-ethereum"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-js-square"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-react"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-node-js"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-npm"
              />
            </li>
            <li
              class="list-inline-item"
            >
              <i
                class="fab fa-git"
              />
            </li>
          </ul>
        </div>
      </section>
    </div>
  </div>
</DocumentFragment>
`;
